packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -961,6 +983,70 @@ export class DataTable {
     }
   }
 
+  /**
+   * getRowActionMenuProps
+   * @param rowData rowData - complete data of the current row
+   * @returns props for kebab menu
+   */
+  getRowActionMenuProps(rowData: DataTableRow) {
+    if (this.showRowActionsAsMenu) {
```

Comment:
you can remove the check for `showRowActionsAsMenu` here. Since this is already handled when invoking `renderKebabMenu`